server/ciphersuites_1.4.go

```diff
@@ -1,33 +0,0 @@
-// Copyright 2015 Apcera Inc. All rights reserved.
```

Comment:
Done...  I didn't add 1.8beta2 compilation for travis, but build and tests pass locally.  Shall I add that at this point, or do we want to wait for the 1.8 release?